examples/csvimport.py

```diff
@@ -49,11 +49,11 @@ async def start(self, ctx: Context) -> None:
             url=f"sqlite:///{db_path}",
             ready_callback=lambda bind, factory: metadata.create_all(bind),
         )
-        await super().start(ctx)
+        await super().start()
```

Comment:
Is it still needed?

Reply:
I need to check why mypy didn't pick this up.

Reply:
Right, I was looking at something else. It's not a syntax or typing issue. I'll do a a sweep on the subprojects to see if this is still present somewhere.

Reply:
I don't think it's wrong to `await super().start()`, just that it does nothing.

Reply:
I found a bunch of similar issues, plus outdated examples where `add_component()` was still called in `start()`.

Reply:
Should be all good here now.